src/app/loadout-builder/generated-sets/CompareDrawer.tsx

```diff
@@ -36,43 +36,12 @@ function getItemStats(item: DimItem, assumeMasterwork: boolean | null) {
     (value) => item.stats?.find((s) => s.statHash === value)?.base || 0
   );
 
-  // Checking energy tells us if it is Armour 2.0 (it can have value 0)
-  if (item.sockets && item.energy) {
```

Comment:
Now that we are only sending in armour 2.0 we can forget the energy check imo but I could be swayed otherwise